widget_driver/lib/src/utils/platform_environment.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:io';
+
+import 'package:flutter/foundation.dart';
+
+extension WebPlatformInfo on Platform {
+  static bool get isWeb => kIsWeb;
+}
+
+typedef GetEnvironment = Map<String, String> Function();
```

Comment:
Maybe name it EnvironmentCallback?

Reply:
Hmm. But what is an "EnvironmentCallback"? Something which the environment calls?
What would you expect it to do?
I could name it "Get EnvironmentCallback". But then it sounds like you get the callback.. 
What do you think?
I will also think about it

Reply:
Okay, I changed it now to be called `EnvironmentCallback` and added a small documentation to it.